jbi/router.py

```diff
@@ -75,6 +76,20 @@ async def bugzilla_webhook(
     return await execute_or_queue(webhook_request, queue, actions)
 
 
+@router.get(
+    "/dl_queue/",
+    dependencies=[Depends(api_key_auth)],
+)
+async def inspect_dl_queue(queue: Annotated[DeadLetterQueue, Depends(get_dl_queue)]):
+    """API for viewing queue content"""
+    bugs = await queue.retrieve()
+    results = defaultdict(list)
+    for bug_id, items in bugs.items():
+        async for item in items:
+            results[bug_id].append(item.model_dump())
+    return results
```

Comment:
Wouldn't this be a good usecase for the unused code we're deleting in https://github.com/mozilla/jira-bugzilla-integration/pull/964/?

Reply:
I find the queue and backend confusing with the current naming and vague docstrings